common/build/buildutils.go

```diff
@@ -461,6 +462,22 @@ func (bc *BuildConfiguration) IsLoadedFromConfigFile() bool {
 	return bc.loadedFromConfigFile
 }
 
+// ResolveBaseModuleName returns the module name to publish build info.
+// If the module name is not specified, it falls back to the current working directory name.
+// If the working directory cannot be retrieved, it defaults to "module".
+// Module name is mandatory to publish build info.
+func (bc *BuildConfiguration) ResolveBaseModuleName() (moduleName string, err error) {
+	if bc.module != "" {
+		moduleName = bc.module
+		return
+	}
+	wd, err := os.Getwd()
+	if err != nil {
+		return
```

Comment:
Let's put here checkerr and remove it from all other users of ResolveBaseModuleName